### PR TITLE
October newsletter draft

### PR DESCRIPTION
**Pull request by bethaitman**

### CODE REVIEW

**Comment by bethaitman** [comment]
@goatlady, any chance you know what the link for the salary survey will be?

File: docs/blog/newsletter-october-2021.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+.. post:: October 05, 2021
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – October 2021
+#########################################
+
+Hello everyone!
+
+It's Beth here, writing while I'm listening in to Write the Docs Prague. I hope everyone who attended had a great time! For those of you who couldn't make it, keep an eye out: we'll be publishing the talk videos on YouTube fairly soon. And if you'd like to get more of a flavour of the conference, you can listen to `the intermission music <https://open.spotify.com/artist/36ZvEAnAbW8NOJK7TrBoh5>`__ - which I am bopping along to as I type :D
```
Comment (bethaitman) [inline docs/blog/newsletter-october-2021.rst]: btw - I liked the idea of people listening to the same music I was while they read, but let me know if you think it's weird!

Comment (thatdocslady) [reply docs/blog/newsletter-october-2021.rst]: That's a really awesome thought! I'm just not sure if Spotify is the most accessible platform for some people... Maybe you can reword to invite people to visit my artist website so they can decide how to consume the music?

Comment (hillaryfraley) [reply docs/blog/newsletter-october-2021.rst]: Very cool idea!

File: docs/blog/newsletter-october-2021.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+You may be able to publish your best efforts without review and await (or seek) corrections, whether from users or others at your company. Docs have bugs just like code, and speedy fixes may be more within your control. The downside with this approach is any mistakes are public until you fix them.
+
+The fastest way to get corrections may be to publish errors, but sometimes that's not an option. In this case, try to assemble an informal group of docs review resources. Seek a product manager or subject matter expert to review docs for accuracy. For readability, look for someone else whose job requires writing, perhaps in marketing or communications. Your experiments here could help drive support for a more formal docs review process.
+
+Some lone writers solicit docs review by creating feedback channels in Slack with the `Workflow Builder <https://slack.com/help/articles/360035692513-Guide-to-Workflow-Builder>`_. For example, you could create a #documentation channel with a built-in form for submitting correction requests, with fields for specifying the page in question and the requested corrections. Then the Slack bot can notify you when someone submits a correction.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2021.rst]:
```suggestion
Some lone writers solicit docs review by creating feedback channels in Slack with the `Workflow Builder <https://slack.com/help/articles/360035692513-Guide-to-Workflow-Builder>`_. For example, you could create a #documentation channel with a built-in form for submitting correction requests, with fields for specifying the page in question and the requested corrections. Then the Slackbot can notify you when someone submits a correction.
```

Comment (hillaryfraley) [reply docs/blog/newsletter-october-2021.rst]: I should have checked this earlier ^ Slack doesn't use a space in "Slackbot"

**Comment by goatlady** [comment]
> @goatlady, any chance you know what the link for the salary survey will be?

https://salary-survey.writethedocs.org/ - we've gone generic, just makes it easier :)

**Review by hillaryfraley — APPROVED** [review]
Looks great!

File: docs/blog/newsletter-october-2021.rst (hunk 1 above)
Comment (bethaitman) [reply docs/blog/newsletter-october-2021.rst]: Sorry, I forget as I use spotify - will switch up the link!

**Pull request merged**